src/algokit_utils/_debugging.py

```diff
@@ -174,8 +175,18 @@ def _build_avm_sourcemap(  # noqa: PLR0913
     return AVMDebuggerSourceMapEntry(str(source_map_output_path), program_hash)
 
 
+@deprecated(
```

Comment:
In the case you want to debug TEAL, this is still useful. I thought we were only planning to remove the creation of the registry file?

Reply:
@neilcampbell probably got sidetracked with the debug package on utils side and overlooked updating py stuff again. Should be addressed now